backend/package-lock.json

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
Actually - we don't need that file. `backend/package.json` is right now kind of artificial, to allow users to import from `backend` folder.

Reply:
Ou, okay 😁 

In that case, I'll remove the `package-lock.json` file 😉 
Thanks for the insight!

Reply:
Done 🎉